app/views/v2/liability_view.scala.html

```diff
@@ -70,11 +76,13 @@
             }
 
             @* activity*@
+            @if(activityViewModel.isDefined) {
```

Comment:
Does that need to be an Option, since this tab is not optional?

Reply:
just for test to be easier to test view itself